frontend/src/components/TransactionList/TransactionItems.js

```diff
@@ -8,13 +8,13 @@ export default class TransactionItems extends React.Component {
         let style = {color: 'black'};
         let transaction = this.props.transaction;
 
-        if (this.props.transaction.type === "Withdrawal") {
+        if (this.props.transaction.TransactionType === "Withdrawal") {
             prefix = "-";
         }
         return (
             <tr>
-                <th >{transaction.type}</th>
-                <th >{transaction.date}</th>
+                <th >{transaction.TransactionType}</th>
```

Comment:
transaction.TransactionType looks like a naming duplication. 

http://blog.thecodewhisperer.com/images/a-model-for-improving-names/improving-names.png